source/_integrations/media_player.markdown

```diff
@@ -48,6 +48,41 @@ Available services: `turn_on`, `turn_off`, `toggle`, `volume_up`, `volume_down`,
 | `entity_id`            |      yes | Target a specific media player. To target all media players, use `all`.                                                                                                                       |
 | `media_content_id`     |       no | A media identifier. The format of this is integration dependent. For example, you can provide URLs to Sonos and Cast but only a playlist ID to iTunes.                   |
 | `media_content_type`   |       no | A media type. Must be one of `music`, `tvshow`, `video`, `episode`, `channel` or `playlist`. For example, to play music you would set `media_content_type` to `music`. |
+| `extra`                |      yes | Extra dictionary data eg. title, thumb send to pychromecast. Possible values can be found below.
```

Comment:
Small tweaks and removed the naming of the underlying library. It doesn't matter in this context.

```suggestion
| `extra`                |      yes | Extra dictionary data to send, e.g., title, thumbnail. Possible values can be found below.
```